node_modules/oae-core/lhnavigation/lhnavigation.html

```diff
@@ -5,7 +5,7 @@
     <div class="oae-lhnavigation">
         <ul class="nav nav-list"><!-- --></ul>
     </div>
-    <div class="oae-page"><!-- --></div>
+    <div id="oae-page-content" class="oae-page"><!-- --></div>
```

Comment:
If there's only one skip link, then it should go to the main content.

[WebAIM](http://webaim.org/techniques/skipnav/):

> provide a link at the top of the page which jumps the user down to an anchor or target at the beginning of the main content.

[Web Accessibility](https://www.webaccessibility.com/best_practices.php?best_practice_id=543):

> The skip link should be located at the beginning of the page and should target the unique content of the page.

Some sites have two (or more) skip links, e.g. one to skip the header and another to skip to content. The text for our skip links has always been "Skip to content."